lib/authority/controller.rb

```diff
@@ -111,7 +109,7 @@ def add_actions(action_map)
       # for all Rails actions in the action map
       def force_action(forced_action)
         add_actions(
-          authority_action_map.transform_values {|_| forced_action }
```

Comment:
when i added rubocop and ran it locally with auto-fix, rubocop changed the code to this line, which is not compatible with older versions of ruby.

undid this change and removed rubocop for the time being. we can revist adding that later.